Library/Formula/ctorrent.rb

```diff
@@ -12,6 +12,11 @@ class Ctorrent < Formula
     sha256 "01c353acf408ba34f5ee8c4ac6acca99ab7cdb285f2e03366d9c18d9e44f221e" => :mountain_lion
   end
 
+  patch do
```

Comment:
Please add a short explanation of what the patch is doing and where it has been submitted upstream, Thanks!